src/oscontainer.py

```diff
@@ -26,6 +26,10 @@
 # https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/builds/custom-builds-buildah
 NESTED_BUILD_ARGS = ['--storage-driver', 'vfs']
 
+# These packages have their versions as labels
+IMPORTANT_PACKAGES = ['kernel', 'kernel-rt-core', 'systemd', 'ignition', 'ostree',
```

Comment:
As a follow-on, perhaps we should remove this hard-coded list and make it configurable in `manifest.yaml` or the like?

Reply:
I think having it in https://github.com/openshift/os/blob/master/oscontainer.yaml would make sense.